package/kernel/linux/modules/netdevices.mk

```diff
@@ -521,7 +521,7 @@ define KernelPackage/igb
   TITLE:=Intel(R) 82575/82576 PCI-Express Gigabit Ethernet support
   DEPENDS:=@PCI_SUPPORT +kmod-i2c-core +kmod-i2c-algo-bit +kmod-ptp
   KCONFIG:=CONFIG_IGB \
-    CONFIG_IGB_HWMON=n \
+    CONFIG_IGB_HWMON=m \
```

Comment:
This is a bool option, please use CONFIG_IGB_HWMON=y
This also adds a dependency to  CONFIG_HWMON which is a module on some targets.

Reply:
Changed to `y`.  For the `igbvf` and `ixgbevf` drivers I've turned it on there as well.